charts/argo-cd/templates/argocd-application-controller/deployment.yaml

```diff
@@ -0,0 +1,355 @@
+{{- if .Values.controller.dynamicClusterDistribution | default false }}
```

Comment:
This default is probably not needed as `dynamicClusterDistribution: false` is defined in values.yaml